### ISSUE TITLE
requestPath.join is not a function

### ISSUE THREAD

**Issue by thatnerdjosh** [bigcommerce/stencil-cli#654]
### Expected behavior
Running `stencil start` would run the page

### Actual behavior
Crashes with error: `TypeError: requestPath.join is not a function`

I recently had the same template running on a windows vagrant box I set up, but running in linux I am getting this error

EDIT (add full stack trace):
```
[Browsersync] Reloading Browsers...
Debug: internal, implementation, error
    TypeError: requestPath.join is not a function
    at Object.internals.getTemplatePath (/home/parallels/.nvm/versions/node/v12.19.0/lib/node_modules/@bigcommerce/stencil-cli/server/plugins/renderer/renderer.module.js:328:40)
    at Object.internals.getPencilResponse (/home/parallels/.nvm/versions/node/v12.19.0/lib/node_modules/@bigcommerce/stencil-cli/server/plugins/renderer/renderer.module.js:366:38)
    at Object.internals.parseResponse (/home/parallels/.nvm/versions/node/v12.19.0/lib/node_modules/@bigcommerce/stencil-cli/server/plugins/renderer/renderer.module.js:221:22)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async internals.implementation (/home/parallels/.nvm/versions/node/v12.19.0/lib/node_modules/@bigcommerce/stencil-cli/server/plugins/renderer/renderer.module.js:39:20)
    at async module.exports.internals.Manager.execute (/home/parallels/.nvm/versions/node/v12.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/@hapi/hapi/lib/toolkit.js:45:28)
    at async Object.internals.handler (/home/parallels/.nvm/versions/node/v12.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/@hapi/hapi/lib/handler.js:46:20)
    at async exports.execute (/home/parallels/.nvm/versions/node/v12.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/@hapi/hapi/lib/handler.js:31:20)
    at async Request._lifecycle (/home/parallels/.nvm/versions/node/v12.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/@hapi/hapi/lib/request.js:312:32)
    at async Request._execute (/home/parallels/.nvm/versions/node/v12.19.0/lib/node_modules/@bigcommerce/stencil-cli/node_modules/@hapi/hapi/lib/request.js:221:9)
```

### Steps to reproduce behavior
Run `stencil start`

**Comment by gabrielsurraco** [comment]
I am having the same issue on mac after updating to the last version of stencil -> https://timeline.noticeable.io/Q8YuGWPOCMmZDwnKUywV/posts/stencil-cli-3-0-release

**Comment by thatnerdjosh** [comment]
Yup same case here, I'll try downgrading if I can

**Comment by thatnerdjosh** [comment]
@gabrielsurraco `npm install -g @bigcommerce/stencil-cli@2.1.1` works for me

**Comment by gabrielsurraco** [comment]
Thanks Josh, Yes, the issue is happening with stencil 3.0 at the moment.

**Comment by MaxGenash** [comment]
@thatnerdjosh thanks for the report.
I'll take a look at what may be causing this issue

**Comment by MaxGenash** [comment]
@thatnerdjosh The fix has just been released as v3.0.1.
Please, try updating the Stencil-CLI again and check if it works fine now for you.

**Comment by thatnerdjosh** [comment]
@MaxGenash it works now... also seems quite a bit faster :) Thanks!
